go.mod

```diff
@@ -94,10 +90,10 @@ require (
 	github.com/sirupsen/logrus v1.6.0
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
-	github.com/spf13/viper v1.4.0
+	github.com/spf13/viper v1.6.2
 	github.com/stoewer/go-strcase v1.2.0
 	github.com/stretchr/testify v1.6.1
-	github.com/tektoncd/pipeline v0.11.3
+	github.com/tektoncd/pipeline v0.14.2
```

Comment:
are you upgrading to tekton 0.14? (just curious)

Reply:
No, but I needed that for something over in Lighthouse, so we end up inheriting that bump here in jx too. Doesn't break anything. =)